styx-flyte-client/src/test/java/com/spotify/styx/flyte/client/TestAdminService.java

```diff
@@ -50,6 +53,24 @@ public void createExecution(final ExecutionOuterClass.ExecutionCreateRequest req
     responseObserver.onCompleted();
   }
 
+  @Override
+  public void getLaunchPlan(final flyteidl.admin.Common.ObjectGetRequest request,
+                            final StreamObserver<LaunchPlanOuterClass.LaunchPlan> responseObserver) {
+    responseObserver.onNext(LaunchPlanOuterClass.LaunchPlan
+        .newBuilder()
+        .setId(request.getId())
+        .setSpec(LaunchPlanOuterClass.LaunchPlanSpec
+            .newBuilder()
+            .setDefaultInputs(Interface.ParameterMap.newBuilder()
+                .putParameters("parameter", Interface.Parameter.newBuilder()
+                    .setDefault(buildLiteralForPartition("2020-09-21"))
```

Comment:
Maybe we don't need to pass any inputs here, or we could pass "styx_parameter" or PARAMETER_NAME constant